test/e2e/assets/kc-e2e-test-repo/packages/pkg.test.carvel.dev/pkg.test.carvel.dev.3.0.0-rc.1.yml

```diff
@@ -9,8 +9,10 @@ spec:
   # PackageInstalls to unless it explicitly allowed within the PackageInstall
   # version selection.
   version: 3.0.0-rc.1
+  releasedAt: 2021-05-05T18:57:06Z
```

Comment:
Since this is for illustrative purposes, would it actually make sense to show this as released later than version 2.0.0?